utils/utils/src/main/java/pl/touk/nussknacker/springframework/util/NumberUtilsConsts.java

```diff
@@ -0,0 +1,5 @@
+package pl.touk.nussknacker.springframework.util;
+
+public class NumberUtilsConsts {
```

Comment:
Let's rename this class to some name that would explain why we do this trick e.g. `BigDecimalScaleEnsurer`, move all setScale invocations and write a notice that this is work around for spel's behaviour with an explanation how spel behaves. Basically let's move our online conversion to comment :)

Reply:
done